test-integ/peering_commontopo/asserter.go

```diff
@@ -3,294 +3,15 @@
 
 package peering
 
-import (
-	"fmt"
-	"io"
-	"net/http"
-	"net/url"
-	"regexp"
-	"testing"
-	"time"
+import "github.com/hashicorp/consul/test-integ/topoutil"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
+// Deprecated: topoutil.Asserter
```

Comment:
I'm not clear why you're doing this deprecation dance. Why not use the real things directly?

Reply:
I did not want to also update every `peering_commontopo` file at the same time. If you think it's warranted I can also inline these moves.

Reply:
It's gotta be done some time. Put it in a separate followup PR or something if you feel like it obscures the intent of this, but as it stands it's just debt